oxd-server/src/main/resources/swagger.yaml

```diff
@@ -54,11 +54,27 @@ paths:
                   - https://client.example.org/cb
                   items:
                     type: string
+                op_configuration_endpoint:
+                  type: string
+                  example: https://op.example.com/acme/.well-known/openid-configuration
+                  description: The openid configuration endpoint URL. If missing, then either
+                    it must be present in defaults or `op_host` must be defined.
                 op_host:
                   type: string
                   description: Provide the URL of OpenID Provider (OP). If missing,
-                    must be present in defaults.
+                    then either it must be present in defaults or `op_configuration_endpoint`
+                    must be defined.
```

Comment:
In description better start with: `Deprecated in favor of `op_configuration_endpoint`. It will be removed in future version(s). `  We should mention about it in all usages.

Reply:
Done